client/src/components/LobbyChat.svelte

```diff
@@ -0,0 +1,73 @@
+<script lang="ts">
+  import Modal from "./Modal.svelte";
+  import { onMount, onDestroy } from "svelte";
+  import { authStore } from "$stores/auth";
+  import { onSnapshot, orderBy, query, where } from "firebase/firestore";
+  import type { LobbyChatMessage, Lobby, Player } from "$lib/firebase/firestore-types/lobby";
+  import type { Unsubscribe, User } from "firebase/auth";
+  import { getLobbyChatCollection } from "$lib/firebase/firestore-collections";
+  import { addLobbyChatMessages } from "$lib/firebase/chat";
+  import ChatMessages from "./ChatMessages.svelte";
+
+  export let lobbyData: Lobby & { id: string };
+
+  let user = $authStore as User;
+  let userInfo: Player;
+  let openSignInModal = false;
+  let errorMessage: string = "";
```

Comment:
you prolly want to display errors somewhere in the html. So this shouldnt be greyed out

Reply:
or remove it completely if the errors are being displayed somewhere else

Reply:
Error messages now display in modal under text input